Src/PerformanceCollector/Shared/PerformanceCollectorModule.cs

```diff
@@ -125,6 +131,12 @@ public void Initialize(TelemetryConfiguration configuration)
                             throw new ArgumentNullException("configuration");
                         }
 
+                        if (!this.EnableIISExpressPerformanceCounters && IsRunningUnderIisExpress())
+                        {
+                            PerformanceCollectorEventSource.Log.RunningUnderIisExpress();
```

Comment:
Maybe only report it in debug output and not send event? Otherwise we just came up with a new startup trace message for every developer using Application Insights.

If we really need this message in portal I'd at least roll a dice and only report it in 30% of times using Random function =)